components/AppLayout/AppLayout.jsx

```diff
@@ -3,7 +3,7 @@ import styles from '@/styles/AppLayout.module.css'
 import Head from 'next/head'
```

Comment:
The nav is broken

<img width="747" alt="Screenshot 2023-03-08 at 10 08 42 PM" src="https://user-images.githubusercontent.com/11657599/223906617-bfdc66c2-d129-4932-8919-0d274ffd3f39.png">

Reply:
Fixed